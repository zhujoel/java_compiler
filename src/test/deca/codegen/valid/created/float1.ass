; start main program
; Class Declaration
; Code de la table des m√©thodes de Object
	LOAD #null, R0
	STORE R0, 1(GB)
	LOAD code.Object.equals, R0
	STORE R0, 16(GB)
; Main Function
; Test pour savoir si la pile est pleine
	TSTO #3
	BOV pile_pleine
	ADDSP #3
; Declaration des variables
	LOAD #0x1.0p0, R2
	STORE R2, 17(GB)
; Beginning of main instructions:
	LOAD 17(GB), R1
	WFLOAT
	HALT
; end main program
