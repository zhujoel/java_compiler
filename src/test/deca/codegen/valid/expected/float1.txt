1.00000e+00
